.azure-pipelines/test_plan.py

```diff
@@ -150,74 +153,81 @@ def parse_list_from_str(s):
             if single_str.strip()]
 
 
+def run_cmd(cmd):
+    process = subprocess.Popen(
+        cmd.split(),
+        shell=False,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE
+    )
+    stdout, stderr = process.communicate()
+    return_code = process.returncode
+
+    if return_code != 0:
+        raise Exception(f'Command {cmd} execution failed, rc={return_code}, error={stderr}')
+    return stdout, stderr, return_code
+
+
 class TestPlanManager(object):
 
-    def __init__(self, scheduler_url, community_url, frontend_url, client_id=None):
+    def __init__(self, scheduler_url, frontend_url, elastictest_msal_client_id, sonic_automation_umi):
```

Comment:
It would be better to make the variable name more generic here:

elastictest_msal_client_id -> client_id
sonic_automation_umi -> managed_identity_id

Reply:
refined.